Improve documentation of numeric groups (#40831)
home-assistant/home-assistant.github.io: source/_integrations/group.markdown

- The group state is combined / calculated based on `type` selected to determine the minimum, maximum, latest (last), mean, median, range, product, standard deviation, or sum of the collected states.
- Members can be any `sensor`, `number` or `input_number` holding numeric states.

> - States which are missing from the state machine does not make the state `unavailable` or `unknown`

Reviewer: ```suggestion
- States which are missing from the state machine do not make the state `unavailable` or `unknown`.
```

- The group state is `unavailable` if no group member has a numeric state.

> - The configuration variable `ignore_non_numeric` controls the behaviour of the group when the group is not `unavailable`:

Reviewer: ```suggestion
- The configuration variable `ignore_non_numeric` controls the behavior of the group when the group is not `unavailable`:
```

>    - When set to `false` (the default), the group state is calculated according to the `type` if all members have a numeric state and set to `unknown` otherwise

Reviewer: ```suggestion
   - When set to `false` (the default), the group state is calculated as follows:
      - if all members have a numeric state: calculated according to the `type` 
      - otherwise: set to `unknown` 
```

>    - When set to `true`, the group state is calculated according to the `type` if at least one member has a numeric state and set to `unknown` otherwise

Reviewer: ```suggestion
   - When set to `true`, the group state is calculated as follows:
      - if at least one member has a numeric state: calculated according to the `type`
      - otherwise:  and set to `unknown`
```

  type: string
  required: true
ignore_non_numeric:

>   description: Only available for `sensor` group. Controls how the [state is calculated when group members have non numeric state](#sensor-number-and-input_number-groups).

Reviewer: ```suggestion
  description: Only available for `sensor` group. Controls how the [state is calculated when group members have non-numeric state](#sensor-number-and-input_number-groups).
```
